### PR TITLE
Clarify Enterprise vs. Non-Enterprise Support

### PR DESCRIPTION
**Pull request by barberscott**
## Description & motivation
Clarify the descriptions of what support offers based on Enterprise vs. Non-Enterprise plan tiers.

### CODE REVIEW

File: website/docs/docs/dbt-support.md [hunk 1]
```diff
@@ -9,40 +9,44 @@ If you're developing in the command line (CLI) and have questions or need some h
...
+    - update user groups.
+- **I need help understanding...**
+    - why this run failed.
+    - why I am getting this error message in dbt Cloud.
+    - why my CI jobs are not kicking off as expected.
+
+
+### dbt Cloud Enterprise accounts
+
+For customers on a dbt Cloud Enterprise plan, we **also** offer basic assistance in troubleshooting issues with your dbt Project.
```
Comment (runleonarun) [inline website/docs/docs/dbt-support.md]:
```suggestion
For customers on a dbt Cloud Enterprise plan, we **also** offer basic assistance in troubleshooting issues with your dbt project.  
```

File: website/docs/docs/dbt-support.md [hunk 2]
```diff
@@ -9,40 +9,44 @@ If you're developing in the command line (CLI) and have questions or need some h
...
-
 Types of questions you should ask your Solutions Architect and Sales Director:
 - How should we think about setting up our dbt projects, environments, and jobs based on our company structure and needs?
 - I want to expand my account! How do I add more people and train them?
 - Here is our data road map for the next year - can we talk through how dbt fits into it and what features we may not be utilizing that can help us achieve our goals?
 - It is time for our contract renewal, what options do I have?
 
 
-When you need help writing SQL or want someone to actually help build your dbt Project, check out our list of [dbt Preferred Consulting Providers](https://www.getdbt.com/ecosystem/) or our [Services](https://www.fishtownanalytics.com/professional-services/) page!
+When you need help writing SQL, reviewing the overall performance of your project, or want someone to actually help build your dbt Project, check out our list of [dbt Preferred Consulting Providers](https://www.getdbt.com/ecosystem/) or our [Services](https://www.fishtownanalytics.com/professional-services/) page!
```
Comment (runleonarun) [inline website/docs/docs/dbt-support.md]:
```suggestion
When you need help writing SQL, reviewing the overall performance of your project, or want someone to actually help build your dbt project, check out our list of [dbt Preferred Consulting Providers](https://www.getdbt.com/ecosystem/) or our [Services](https://www.getdbt.com/dbt-labs/services/) page!
```

**Review by runleonarun — DISMISSED** [review]
@barberscott This looks good! Just some tiny suggestions. We can get it merged once those are included!

**Comment by barberscott** [comment]
Added changes.

**Review by runleonarun — APPROVED** [review]
:shipit: !

**Pull request merged**